app/controllers/policy_classes_controller.rb

```diff
@@ -4,7 +4,9 @@ class PolicyClassesController < PlanningApplicationsController
   before_action :set_planning_application
   before_action :set_policy_class, only: %i[show update destroy]
 
-  def part; end
+  def part
+    @part_number = params[:part]&.to_i
+  end
```

Comment:
This is just a fix for a small bug I found as I was clicking around. Currently if you click 'Add assessment area', select a part, click 'Continue' and then click 'Back', the part you selected is no longer selected.